### PR TITLE
Adds update_package_properties and build_packages pipelines into pg-auto-failover

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

**Comment by hanefi** [comment]
@gurkanindibay can you check the failed Ubuntu Xenial tests? Let me remind you that it is EOL'd and we can and maybe we should remove it from our build matrix.

**Comment by gurkanindibay** [comment]
> @gurkanindibay can you check the failed Ubuntu Xenial tests? Let me remind you that it is EOL'd and we can and maybe we should remove it from our build matrix.

I checked the failed test and it's kinda weird. it's about python version that is ubuntu xenial image using. But it does not fail on Travis

File: .github/workflows/build-package.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - el/7
+          - el/8
+          - ol/7
+          - debian/stretch
+          - debian/buster
+          - ubuntu/xenial
```
Comment (hanefi) [inline .github/workflows/build-package.yml]: I think we should remove this line

```suggestion
```

Comment (hanefi) [reply .github/workflows/build-package.yml]: similarly we can update `.travis.yml`, preferably in a separate PR.

Comment (gurkanindibay) [reply .github/workflows/build-package.yml]: Asking @DimCitus might be useful

**Review by hanefi — APPROVED** [review]

**Pull request merged**